lib/app/handler/handler.go

```diff
@@ -1088,5 +1118,25 @@ for BINARY in tele tsh; do
     rm $BINARY
 
     echo "Done! Try running '$BINARY version'"
-done`))
+done
+
+{{ if .tfVersion }}
+mkdir -p ${HOME}/.terraform.d/plugins
+TF_URL=https://get.gravitational.io/telekube/bin/{{.tfVersion}}/$OS/$ARCH
+for BINARY in terraform-provider-gravity terraform-provider-gravityenterprise; do
```

Comment:
I think that the provider binary name should just be `terraform-provider-gravity` and whether it's enterprise or not is determined by how it was compiled. Same as we don't have two binaries, "gravity" and "gravityenterprise", we only have "gravity" which may be of enterprise or OSS edition.

Reply:
Also, OSS provider should get published to hub.gravitaitonal.io (https://github.com/gravitational/ops/tree/master/hub), not get.gravitational.io. That's where all OSS artifacts go.